### PR TITLE
Only use 'antiword' in 'astextplain' if 'file' identified a Word document

### PR DESCRIPTION
**Pull request by Vampire**
Without this change you always get output that a given file is not a Word document, for example during pickaxeing.

### CODE REVIEW

File: git-extra/astextplain [hunk 1]
```diff
@@ -12,7 +12,14 @@ case "$1" in
 		odt2txt "$1" || cat "$1"
 		;;
 	*.doc | *.DOC | *.dot | *.DOT)
-		out=$(antiword -m UTF-8 "$1") && sed "s/\^M$//" <<<$out || cat "$1"
+		case "$(file --brief --mime-type "$1")" in
```
Comment (dscho) [inline git-extra/astextplain:15]: My first reaction was to be concerned about the time spent on spawning `file`, but [`file` has hard enough of a time to identify all the Word Document flavors](https://github.com/search?q=repo%3Afile%2Ffile%20msword&type=code) that the alternative (namely, to hand-roll a script to identify Word Documents) seems prohibitive.

My next reaction was to be concerned about MinGit, but then I looked and found out that MinGit does not ship with `astextplain` (nor with `file`, for that matter).

**Review by dscho — APPROVED** [review]

**Comment by dscho** [comment]
@Vampire thank you for your contribution!

**Comment by S3od11** [comment]
'

**Review by S3od11 — COMMENTED** [review]
'

**Comment by S3od11** [comment]
'

File: git-extra/astextplain (hunk 1 above)
Comment (S3od11) [reply git-extra/astextplain:15]: '

**Comment by S3od11** [comment]
'

**Comment by MayraPeralesContrera** [comment]
Sensores

**Pull request merged**